### DISCUSSION TITLE
Closing commit split after commiting?

### DISCUSSION THREAD

**Discussion by Dich0tomy** [NeogitOrg/neogit#1388] [Q&A]
Hello.
I'm on `v1.0.0` and I noticed that after doing `cc` and finishing the commit an empty vertical split is still present on my screen. I'm running the default configuration with minor changes to the keymaps and nothing else.

https://github.com/NeogitOrg/neogit/assets/75364377/15ad7007-c64b-4ab2-9f50-73e7b01b5346

What can I do about it?

**Comment by CKolkey** [comment]
It was a minor issue with the buffer closing logic. I think it should be fixed now?
